AppIntro.js

```diff
@@ -455,4 +478,9 @@ AppIntro.defaultProps = {
   doneBtnLabel: 'Done',
   skipBtnLabel: 'Skip',
   nextBtnLabel: '›',
+  defaultIndex: 0,
+  showSkipButton: true,
+  showDoneButton: true,
+  paginationStyles: {},
+  dotStyles: {}
```

Comment:
I think `paginationStyles `、`dotStyles ` is not needed , because last someone contribution `customStyles`  https://github.com/FuYaoDe/react-native-app-intro/pull/15,  You can use `customStyles` custom every style, But no written readme It is time to update the readme....